docs/src/installation.md

```diff
@@ -0,0 +1,22 @@
+# Installation
+
+Ultimately we aim to make installing Ray.jl as simple as running `Pkg.add("Ray")`. However, at the moment there are some manual steps are required to install Ray.jl. For users with machines using the Linux x86_64 or macOS aarch64 (Apple Silicon) platforms we have provided pre-built binaries for each Ray.jl release.
+
+To install these dependencies and Ray.jl run the following:
+
+```sh
+# Install the Ray CLI
+PYTHON=$(python3 --version | perl -ne '/(\d+)\.(\d+)/; print "cp$1$2-cp$1$2"')
+case $(uname -s) in
+    Linux) OS=manylinux2014;;
+    Darwin) OS=macosx_13_0;;
+esac
+ARCH=$(uname -m)
+RELEASE="latest/download"  # Alternatively "download/ray-2.5.1+1"
+pip install -U "ray[default] @ https://github.com/beacon-biosignals/ray/releases/$RELEASE/ray-2.5.1-${PYTHON}-${OS}_${ARCH}.whl" "pydantic<2"
+
+# Install the Julia packages "ray_julia_jll" and "Ray"
+julia -e 'using Pkg; Pkg.add([PackageSpec(url="https://github.com/beacon-biosignals/Ray.jl", subdir="ray_julia_jll"), PackageSpec(url="https://github.com/beacon-biosignals/Ray.jl")])'
```

Comment:
won't this add the latest commit? We may want to think about this as the HEAD commit might not have Artifacts associated with it (I just tested this workflow now). 

So we may want to do one of two things:  
a) Create a `dev` branch where we push our active development work, until we create Artifacts at which point we merge into `main`
b) Branch off `main` for each proxy "release" that has artifacts and add `rev=<stable-branch>`

B is easier short-tem but these instructions will get out of date, whereas `a` is more painful short term but it would lead to more stable release cycles...

Reply:
This does pull down the latest `main`. Your points are valid here and I think the best course of action is to just specify using the `v0.1.0` tag for now. That doesn't guarantee stability but that is probably want we want to recommend people use until we get our first release done. These instructions are just meant for now until we get a proper release.

Additionally, the same issue exists with the Ray CLI as we need to ensure that version is compatible with the Ray.jl version.